src/Billing/InsurancePolicyTypes.php

```diff
@@ -18,7 +18,7 @@ class InsurancePolicyTypes
 {
     // Hard-coding this array because its values and meanings are fixed by the 837p
 // standard and we don't want people messing with them.
-    const POLICY_TYPE_NOT_APPLICABLE = '';
+    const POLICY_TYPE_NOT_APPLICABLE = 'FALSE';
```

Comment:
So do we want to actually save a FALSE value in the database for this option?  Will this break legacy database entries?  You'd know this better than I do, just a bit confused on how we get to the FALSE option.

Reply:
was working on some strange data, after looking at demo data it appears the policy type is saved as an empty string, then thought maybe on upgrade we should change empty string to null and allow null to be `Yes`
![Screenshot from 2024-04-09 17-45-21](https://github.com/openemr/openemr/assets/5618372/c79c9cc4-5408-478b-9058-d4a08271cb81)
![Screenshot from 2024-04-09 17-47-33](https://github.com/openemr/openemr/assets/5618372/25d20486-1a57-45c1-873d-cb0883bef6d6)

Reply:
then thought this is pretty trivial :smile: